.github/workflows/doctests.yml

```diff
@@ -0,0 +1,80 @@
+name: Doctests
+
+on:
+  push:
+    branches:
+      - doctest*
+  repository_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
+
+env:
+  HF_HOME: /mnt/cache
+  RUN_SLOW: yes
+  OMP_NUM_THREADS: 16
+  MKL_NUM_THREADS: 16
+
+jobs:
+  run_doctests:
+    runs-on: [single-gpu, nvidia-gpu, a10, ci]
+    container:
+      image: huggingface/diffusers-all-latest-gpu
```

Comment:
What's the container? Has everything installed like Flax, PyTorch, etc.?

Reply:
cc: @stevhliu Is this just a container with all dependencies installed?

Reply:
Yeah I believe so!